wsi/drm/drmdisplay.cpp

```diff
@@ -65,6 +66,7 @@ bool DrmDisplay::InitializeDisplay() {
   GetDrmObjectProperty("GAMMA_LUT", crtc_props, &lut_id_prop_);
   GetDrmObjectPropertyValue("GAMMA_LUT_SIZE", crtc_props, &lut_size_);
   GetDrmObjectProperty("OUT_FENCE_PTR", crtc_props, &out_fence_ptr_prop_);
+  GetDrmObjectProperty("background_color", crtc_props, &canvas_color_prop_);
```

Comment:
to align with rest of properties should this be BACKGROUND_COLOR ?

Reply:
I did think about the lower case name but I am not sure if we can count on the fact the kernel driver always exposes property names that are upper case. For example, there are properties like audio, type whose names are exposed in lower case. Alternatively, do you think we should make GetDrmObjectProperty ignore the case of the property name to accommodate lower case property names?

Reply:
this was just a quesiton, I dont see any restriction on having lower/higher case https://01.org/linuxgraphics/gfx-docs/drm/drm-kms-properties.html seems mixed here. Not sure how it's decided